src/deprecated.tex

```diff
@@ -277,7 +277,7 @@ \section{Typesetting Quotations}
 }
 
 The \TeX{} version of entering quotes was to rely on ligatures for the given
-quotation mark. This method is not context aware and cannot be customised.
+quotation mark. This method is not context aware and cannot be customized.
```

Comment:
I used ltex with britsh english settings and it suggested to replace some of the -sed with -zed since they were ok according ... but I am no native speaker either, so ... :)

I agree a style guide would be a good idea.

Reply:
Yes, the British spelling comes from the fact that I was working in Leicester when I did the initial translation and all the friends who proofread were from there ... :)

I think moving to American spelling would make sense.